CHANGELOG.md

```diff
@@ -48,6 +48,11 @@ and this project adheres to
   the UFFD Unix domain socket open to prevent the race condition between the
   guest memory mappings message and the shutdown event that was sometimes
   causing arrival of an empty message on the UFFD handler side.
+- [#5143](https://github.com/firecracker-microvm/firecracker/pull/5143): Fixed
+  to report `process_startup_time_us` and `process_startup_time_cpu_us` metrics
+  for `api_server` right after the API server starts, while previously reported
+  before applying seccomp filter and starting the API server. Users may observe
+  a bit longer startup time metrics.
```

Comment:
Didn't we deprecate those metrics and parameters? if so I think we should not change the meaning.

Reply:
Did we also deprecated metrics as well? I thought that was only for parameters: https://github.com/firecracker-microvm/firecracker/blame/45fc21bcddc17212c22d639096f81ff97c9a344b/DEPRECATED.md#L17-L18

Reply:
As per our past discussion, it seems whether to deprecate the metrics or not is still a room to discuss. At least we're still using it to monitor startup time performance.